deploy/docker/edumfa.py

```diff
@@ -0,0 +1,34 @@
+from os import getenv
+
+
+def _getenv(key, default="") -> str:
```

Comment:
```suggestion
def _getenv(key: str, default: str | None = None) -> str:
```
Added type hints and `None` as default seems more appropriate.

Reply:
done